crates/ln-dlc-node/src/tests/onboard_from_lnd.rs

```diff
@@ -53,6 +53,14 @@ async fn onboard_from_lnd() {
         )
         .unwrap();
 
+    assert!(
+        payer
+            .is_channel_active(coordinator.info.pubkey)
+            .await
+            .unwrap(),
+        "Channel is not active"
+    );
```

Comment:
:question: Was this helpful when debugging? As in, did it ever fail here? If so we can keep it, but I'm keen on keeping the test as simple as possible.

Reply:
the payment was failing, and I wanted make sure it's not the case. I didn't bother with rolling it out in other tests, but I think it's good to check in at least one that the channel gets opened correctly.

Reply:
> I think it's good to check in at least one that the channel gets opened correctly.

This check is already happening when you call `open_channel` on line 24.

Reply:
https://github.com/get10101/10101/blob/51bb3c36fe2f906dc9f035cbcebe4fab5a42e3ea/crates/ln-dlc-node/src/tests/lnd.rs#L80-L94

Reply:
thank you for pointing me to it - I'll remove the unnneeded patch :)